pandas/core/arrays/base.py

```diff
@@ -1161,7 +1161,7 @@ def _create_method(cls, op, coerce_to_dtype=True):
         --------
         Given an ExtensionArray subclass called MyExtensionArray, use
 
-        >>> __add__ = cls._create_method(operator.add)
+        >>> __add__ = cls._create_method(operator.add)  # doctest: +SKIP
```

Comment:
what goes wrong here without the skip?

Reply:
This example is not runnable example, my fix was to add the ```# doctest: +SKIP```, but as @jorisvandenbossche suggested [here](https://github.com/pandas-dev/pandas/pull/33179#discussion_r401543359) removing the ```>>>``` (IMO) is a better solution